src/dbt/kipptaf/models/reporting/intermediate/int_reporting__student_filters.sql

```diff
@@ -75,6 +75,31 @@ with
             end as iready_proficiency,
         from {{ ref("base_iready__diagnostic_results") }}
         where rn_subj_round = 1 and test_round = 'EOY'
+    ),
+
+    composite_only as (
+        select
+            cast(left(bss.school_year, 4) as int) as academic_year,
+            bss.student_primary_id as student_number,
+            'DIBELS Benchmark' as assessment_type,
+            bss.benchmark_period as mclass_period,
+            u.measure as mclass_measure,
```

Comment:
looks like you can remove `assessment_type ` and `mclass_measure `

Reply:
True, true. Done!